openedx/features/subscriptions/api/v1/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+
+from openedx.features.subscriptions.models import UserSubscription
+from enrollment.serializers import CourseEnrollmentSerializer
+
+
+class UserSubscriptionSerializer(serializers.ModelSerializer):
+    """
+    "UserSubscripion" serializer.
+    """
+    course_enrollments = CourseEnrollmentSerializer(many=True, required=False)
```

Comment:
`required=False` would filter out `course_enrollments` which means UserSubscription object will be created without `course_erollments`. Shouldn't we restrict it?

Reply:
@iam-mhaseeb This is for when a user buys the subscription. at that point, a user won't have any enrollments. This record is created through subscription fulfillment module which runs when you buy a new subscription. Subscription buy flow and enroll through subscription flows are different.